cardano_node_tests/tests/kes.py

```diff
@@ -182,20 +185,20 @@ def check_kes_period_info_result(  # noqa: C901
     return errors
 
 
-def get_xfails(errors: List[str]) -> Set[str]:
-    """Get xfail error strings."""
-    xfails = set()
+def get_known_fails(errors: List[str]) -> Set[str]:
```

Comment:
What you are getting here are really XFAILs :-) So there's no need to rename this.